test/LootBoxController.test.js

```diff
@@ -56,6 +56,12 @@ describe('LootBoxController', () => {
     let lootBoxControllerResult = await deployments.get('LootBoxController')
     lootBoxController = await buidler.ethers.getContractAt('LootBoxController', lootBoxControllerResult.address, wallet)
 
+
+  // insert equivilent erc20mintable, erc1155 erc721
+  let ERC20Mintable
+  let ERC20MintableResult = await deployments.get('ERC20Mintable')
+  ERC20Mintable = await buidler.ethers.getContractAt('ERC20Mintable', ERC20MintableResult.address, wallet)
```

Comment:
What is the `ERC20Mintable` var used for?  Perhaps you meant to update the `deployContracts` above

Reply:
Yes, this code should have been removed/serves no purpose.